Pipfile

```diff
@@ -21,7 +21,10 @@ flask-bcrypt = "*"
 flask-jwt-extended = "*"
 marshmallow = "*"
 flask-cors = "*"
+gunicorn = "*"
+python-dotenv = "*"
 werkzeug = "*"
```

Comment:
I know this isn't part of this PR, but I saw it previously and was confused why this is needed? I know it's the base for Flask, but isn't it an implicit dependency?

Reply:
it has something to do with the WSGI (which I don't know much about) but it might not be necessary. I will double-check that before I merge this